src/watch_test.ts

```diff
@@ -34,81 +33,26 @@ const fakeConfig: {
     ],
 };
 
-// Object replacing real Request object in the test
-class FakeRequest extends EventEmitter implements RequestResult {
-    pipe(stream: Duplex): void {}
-    abort() {}
-}
+const systemUnderTest = (): [nock.Scope] => {
+    const scope = nock(fakeConfig.clusters[0].server);
+    return [scope];
+};
 
 describe('Watch', () => {
-    describe('DefaultRequest', () => {
```

Comment:
I'm a little worried that we're deleting these two tests, are they really not needed?

Reply:
those 2 are not relevant anymore as we don't use the DefaultRequest class anymore